src/components/AccordionList.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
+import { useTestIds } from "..";
+import { Accordion, AccordionProps, AccordionSize } from "./Accordion";
+
+interface AccordionListProps {
+    accordions: AccordionProps[];
+    /** Allows multiple accordions to be expanded simultaneously (enabled by default) */
+    allowMultipleExpanded?: boolean;
+    size?: AccordionSize;
+}
```

Comment:
**suggestion** Appears as though we have 4spaces for our tabs, where we should only have 2. Can you run `yarn format` on your workspace?